src/transactionsFactories/transferTransactionsFactory.ts

```diff
@@ -49,14 +49,19 @@ interface IGasEstimator {
 
 /**
  * Use this class to create transactions for native token transfers (EGLD) or custom tokens transfers (ESDT/NTF/MetaESDT).
- * This name is only temporary, the class will be renamed to `TransferTransactionsFactory`.
  */
 export class TransferTransactionsFactory {
     private readonly config?: IConfig;
     private readonly dataArgsBuilder?: TokenTransfersDataBuilder;
     private readonly tokenComputer?: ITokenComputer;
     private readonly gasEstimator?: IGasEstimator;
 
+    /**
+     * Should be instantiated using `Config` and `TokenComputer`.
+     * Instantiating this class using GasEstimator represents the legacy version of this class.
+     * The legacy version contains methods like `createEGLDTransfer`, `createESDTTransfer`, `createESDTNFTTransfer` and `createMultiESDTNFTTransfer`.
+     */
+    // this was done to minimize breaking changes in client code
```

Comment:
Somehow rogue comment. Can be incorporated into the above one (to avoid any negative impact in some IDEs).

Reply:
Incorporated it in the above comment